tests/lapi/lib.lua

```diff
@@ -0,0 +1,52 @@
+--[[
+SPDX-License-Identifier: ISC
+Copyright (c) 2023-2025, Sergey Bronnikov.
+
+Test helpers.
+]]
+
+-- The function determines a Lua version. The function does not
+-- use any standard library function or any global variable,
+-- it does not rely on the name of "_ENV" upvalue (as it can be
+-- renamed when Lua is being built).
+-- The source is [1], see also [2].
```

Comment:
I believe there is much more easier and bulletproof methods to determine
the version. Let's use `jit` and `jit.version` plus `_VERSION` variable.
Also, the 5.4 isn't covered here.

Reply:
Fixed.